CompaniesApp/views.py

```diff
@@ -60,7 +63,7 @@ def joinCompany(request):
 
         return redirect_with_param(request, "company", in_company.name, 'Success! Joined company: '+in_company.name)
     return render(request, 'autherror.html')
-    
+
 def unjoinCompany(request):
```

Comment:
Ah, my mistake, great point. While scrolling around I thought this was unjoinGroup(request).

I love that you found this update_index method, this will make the demo tomorrow so much better!